src/client/app/utils/helperFunctions.js

```diff
@@ -100,6 +100,24 @@ const dateSortDesc = (ary) => {
     return [...ary].sort((a, b) => new Date(b.playedAt) - new Date(a.playedAt));
 };
 
+const playlistUpdateMessage = (messageKey) => {
```

Comment:
Much better!
I think we tighten it up even more: the assignment of `userMessage` is unnecessary since its value can just be returned straight away. The noMatch key is also an unnecessary key/value pair since it's only ever accessed statically. Might as well make it its own assignment. Lastly, the strict equality check is fine but we can achieve the same result with a more terse expression `!userMessages[messageKey]`. Here's what all these suggestions look like:
```
const playlistUpdateMessage = (messageKey) => {
    const userMessages = {
        playlistDate: 'The playlist date was updated',
        showId: 'This playlist has been reassigned to a different show',
        playlistAlreadyExists: 'That playlist already exists',
        playlistDeleteFail: 'Playlist delete failed',
        songUpdated: 'Song successfully updated',
        noChange: 'The document was unchanged',
        noSuccess: 'Update was not successful'
    };
    const noMatch = 'The playlist record has been updated';

    return !userMessages[messageKey] ? noMatch : userMessages[messageKey];
};
```